### DISCUSSION TITLE
Newbi question on security

### DISCUSSION THREAD

**Discussion by manforrilla** [philosowaffle/peloton-to-garmin#350] [Q&A] [answered]
I read the Wiki and followed all the steps to do it all on github, all this is new to me, are the secrets viewable to the public? Like can anyone see my email and passwords? they are just for my health accounts but just curious, I know nothing about all this other than its epic and sooo sooo awesome to finally get all my data into garmin. Thank you all sooo much!!!!!

**Comment by philosowaffle** [comment, marked as answer]
Hi, glad to hear you're excited about P2G!

Secrets added to your Fork via the Secrets dashboard are private and only visible to you.  They will not be public or accessible by anyone else.

**Reply by manforrilla** [reply]
Thank you soo much! Again this is incredible, ive been waiting for years lol.
Eric
